docs/products/postgresql/howto/optimize-pg-slow-queries.md

```diff
@@ -1,13 +1,43 @@
 ---
-title: Optimize PostgreSQL® slow queries
+title: Optimize Aiven for PostgreSQL® slow queries
 ---
 
-PostgreSQL® gives you the ability to
-[indentify slow queries](/docs/products/postgresql/howto/identify-pg-slow-queries) using the `pg_stat_statements` view. The following article
-contains several common optimization techniques to improve slow running
-queries.
+import Insights from "@site/static/images/icons/performance.svg";
+import ActionsIcon from "@site/static/images/icons/more.svg";
 
-## Common slow queries optimizations
+Optimize slow queries to ensure maximum service performance. Use Aiven's AI capabilities or  `pg_stat_statements` to identify optimization opportunities.
+
+## Artificial intelligence recommendations
+
+Use Aiven's artificial intelligence (AI) capabilities to suggest optimizations to
+your databases and queries.
+
+Aiven considers various aspects to suggest optimization, for example query
+structure, table size, existing indexes and their cardinality, column types and
+sizes, the connections between the tables and columns in the query.
+
+### Display optimization recommendations
+
+To optimize a query:
+
+1. In the [Aiven Console](https://console.aiven.io/login), open your Aiven for PostgreSQL service.
+1. Click <Insights className="icon"/> **AI insights**.
+1. For the query of your choice, click **Optimize**.
+   A window displays the query.
+1. In the **Query optimization report** window, see the optimization suggestion.
```

Comment:
This step is not actionable, let's remove it. The info included in this step is moved to the commentary to the previous step.